rust/agama-network/src/nm/dbus.rs

```diff
@@ -261,12 +261,25 @@ pub fn merge_dbus_connections<'a>(
     original: &'a OwnedNestedHash,
     updated: &'a NestedHash,
 ) -> Result<NestedHash<'a>, zvariant::Error> {
+    // FIXME: it should contain all the sections and attributes which can be absent and known by
+    // agama in order to allow the user to remove them otherwise the original value will be kept
```

Comment:
is it really intentional to allow user remove attributes via agama that are not managed by agama? I would just keep it outside of agama as long as agama does not support it.

Reply:
No, that is not possible. That is why we are getting connection values from DBUS in ordert to not lost the values Agama does not know about. 

But the current problem are optional values which are not present in the updated connection but are in the original connection (obtained from DBUS). For example, changing from **interface-name** to **macAddress** was not allowed because both values are optional in the connection and once written to NM there will be obtained when retrieved.